### PR TITLE
rolecode as employee replaced for rp-services and RP_APPROVER is removed

### PR DESCRIPTION
**Pull request by Transerveshivam**

### CODE REVIEW

File: data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json [hunk 1]
```diff
@@ -12822,22 +12822,6 @@
       "leftIcon": "editor:insert-chart",
       "path": "Finance.Set-up.Budget Control Type.Create/Update-Budget Control Type"
     },
-    {
-      "id": 1766,
```
Comment (rajiv-transerve) [inline data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json]: Why is this removed??

**Review by rajiv-transerve — CHANGES_REQUESTED** [review]

File: data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json (hunk 1 above)
Comment (Transerveshivam) [reply data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json]: that is removed because that id belongs to RP_APPROVER

Comment (rajiv-transerve) [reply data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json]: This action is the reason inbox shows up for all users. 
This is not just used by RP_APPROVER but by most of the employees.

**Review by rajiv-transerve — CHANGES_REQUESTED** [review]

File: data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json (hunk 1 above)
Comment (Transerveshivam) [reply data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json]: ok rolling back

**Review by rajiv-transerve — APPROVED** [review]
